app/src/main/java/org/jellyfin/androidtv/details/EpisodeDetailsFragment.kt

```diff
@@ -33,23 +38,37 @@ class EpisodeDetailsFragment(private val data: Episode) : BaseDetailsFragment()
 			// Attach your media item details presenter to the row presenter:
 			val detailsDescriptionPresenter = DetailsDescriptionPresenter(title = data.name, subtitle = "TODO", body = data.description)
 
+			val overviewRowPresenter = FullWidthDetailsOverviewRowPresenter(
+				detailsDescriptionPresenter,
+				primaryImageBitmap?.let { DetailsOverviewLogoPresenter() })
+
+			overviewRowPresenter.onActionClickedListener = OnActionClickedListener {
+				val action = it as BaseAction
```

Comment:
Maybe add a typecheck if it actually is a BaseAction (just to be sure)

Reply:
Will do